ios/Capacitor/Capacitor/CAPBridgeViewController.swift

```diff
@@ -89,15 +89,21 @@ public class CAPBridgeViewController: UIViewController, CAPBridgeDelegate, WKScr
     }
     webView?.configuration.preferences.setValue(true, forKey: "allowFileAccessFromFileURLs")
     view = webView
-    
+
     setKeyboardRequiresUserInteraction(false)
-    
+
     bridge = CAPBridge(self, o, capConfig, specifiedScheme)
 
+    webView?.isOpaque = false
     if let backgroundColor = (bridge!.config.getValue("ios.backgroundColor") as? String) ?? (bridge!.config.getValue("backgroundColor") as? String) {
-      webView?.backgroundColor = UIColor(fromHex: backgroundColor)
-      webView?.scrollView.backgroundColor = UIColor(fromHex: backgroundColor)
+        if #available(iOS 13.0, *) {
+            webView?.backgroundColor = UIColor.systemBackground
```

Comment:
this will break the background color functionality using the systemBackground color on iOS 13 despite the value used